integration_tests/src/main/python/date_time_test.py

```diff
@@ -190,14 +186,12 @@ def test_to_unix_timestamp_improved(data_gen):
         (StringGen('[0-9]{4}/[01][12]/[0-2][1-8]'),'yyyy/MM/dd'),
         (ConvertGen(DateGen(nullable=False), lambda d: d.strftime('%Y/%m').zfill(7), data_type=StringType()), 'yyyy/MM')]
 
-@incompat
 @pytest.mark.parametrize('data_gen,date_form', str_date_and_format_gen, ids=idfn)
 def test_string_to_unix_timestamp(data_gen, date_form):
-    print("date: " + date_form)
+    conf = {"spark.rapids.sql.improvedTimeOps.enabled": "true"}
```

Comment:
The test fails if I don't add this config and I don't understand why yet. I am debugging this still.